dashboard/models.py

```diff
@@ -14,36 +14,28 @@
 import logging
 logger = logging.getLogger(__name__)
 
-from datetime import datetime
 from model_utils import Choices
 
-class AcademicTermsQuerySet(models.QuerySet):
-    def course_date_start(self, course_id):
-        try:
-            return self.get(course__id=str(course_id)).date_start
-        except self.model.DoesNotExist:
-            logger.debug(f"Could not find term for course {course_id}")
-            return datetime.min
-
-class AcademicTermsManager(models.Manager):
-    def get_queryset(self):
-        return AcademicTermsQuerySet(self.model, using=self._db)
-
-    def course_date_start(self, course_id):
-        return self.get_queryset().course_date_start(course_id)
 
 class AcademicTerms(models.Model):
     id = models.BigIntegerField(primary_key=True, verbose_name="Term Id")
     canvas_id = models.BigIntegerField(verbose_name="Canvas Id")
     name = models.CharField(max_length=255, verbose_name="Name")
-    date_start = models.DateTimeField(verbose_name="Start Date", blank=True, null=True)
-    date_end = models.DateTimeField(verbose_name="End Date", blank=True, null=True)
-
-    objects = AcademicTermsManager()
+    date_start = models.DateTimeField(verbose_name="Term Start Date and Time", blank=True, null=True)
+    date_end = models.DateTimeField(verbose_name="Term End Date and Time", blank=True, null=True)
 
     def __str__(self):
         return self.name
 
+    # # Replace the year in the end date with start date (Hack to get around far out years)
+    # # This should be replaced in the future via an API call so the terms have correct end years, or Canvas data adjusted
+    def get_correct_date_end(self):
+        if (self.date_end.year - self.date_start.year) > 1:
+            logger.debug(f'{self.date_end.year} - {self.date_start.year} greater than 1 so setting end year to match start year.')
```

Comment:
Since the date_end field is changed, the logging level will need to be "info" at least, instead of "debug"

Reply:
Thanks for this correction! I'm still getting comfortable with the `logging` module. See latest commit.